common/tests/mesh_com_1.0/test_mesh_iperf.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+source ./common/common.sh
+
+test_case="mesh_com v1.0 IPERF Traffic Measurements"
+
+description="mesh_com v1.0 Iperf Traffic Throughput Measurements"
+#######################################
+# Init
+# Globals:
+# results
+# device_list
+# Arguments:
+#######################################
+_init() {
+  test_mesh_com_1.0_dependencies
+  _deinit
+
+  count=`python neighbour_count.py`
+  if [ $count -gt 0 ]
+  then
+     echo "Neigbours Exist and there count is : $count"
+  else
```

Comment:
try to use a unified pattern,  (words with capital or not )

Reply:
@martin-tii Issues resolved